packages/api/test/resolvers/subscriptions.test.ts

```diff
@@ -347,4 +348,91 @@ describe('Subscriptions API', () => {
       await getRepository(Subscription).remove(subscription)
     })
   })
+
+  describe('Subscribe API', () => {
+    const query = (
+      name: string | null,
+      url: string | null,
+      subscriptionType: SubscriptionType | null
+    ) => `
+      mutation {
+        subscribe(input: {
+          name: ${name ? `"${name}"` : null} 
```

Comment:
its probably ok in some tests but I've noticed its not good to use string interop in the GQL as it can give really unexpected results sometimes like maybe an empty string for something that was a null value.

Reply:
Yeah, I was hesitated to write this 😞 Gonna change it for sure